web/modules/custom/dpl_admin/dpl_admin.module

```diff
@@ -216,7 +216,54 @@ function _dpl_admin_form_alter_node_page(array &$form, FormStateInterface $form_
     $form['field_hero_title']['#states']['invisible'] = $inactive_state;
     $form['field_subtitle']['#states']['invisible'] = $inactive_state;
   }
+}
 
+/**
+ * Implements hook_form_FORM_ID_alter() for the user add/edit form.
+ *
+ * Re-ordering the user edit form, to make the password demands clearer.
+ *
+ * @param array<mixed> $form
+ *   See the $form in dpl_admin_form_alter().
+ */
+function _dpl_admin_form_alter_user_form(array &$form, FormStateInterface $form_state, string $form_id): void {
+  $form['account']['current_pass']['#weight'] = 997;
+  $form['account']['pass']['#weight'] = 998;
+  $form['account']['password_policy_status']['#weight'] = 999;
```

Comment:
Please consider adding a comment about why these (very high) numbers are set.

When setting weights it is usually in relation to another value. Perhaps you just want these fields to appear last?